Externals/expr/include/expr.h

```diff
@@ -934,7 +966,7 @@ extern "C" {
         free(e->param.func.context);
       }
     }
-    else if (e->type != OP_CONST && e->type != OP_VAR) {
+    else if (e->type != OP_CONST && e->type != OP_VAR && e->type != OP_STRING) {
```

Comment:
There should be a separate case to free the string; this is just leaking it.

Reply:
I don't know how to write this case out, if you could help.

Reply:
You can just revert this line and add the following above it:
```
} else if (e->type == OP_STRING) {
  free(e->param.str.s);
}
```

Reply:
`Error	C2664	'void free(void *)': cannot convert argument 1 from 'const char *' to 'void *'`

Will this not be handled in expr_destroy()? Is there a reason why expr_destroy_args needs to handle STRING but not CONST or VAR?

Reply:
You probably want to declare str::s as a char * rather than const char *. CONST/VAR expressions don't make separate allocations that are owned by the expr struct. VAR expressions do store a pointer, but the ownership of the memory belongs to expr_var_list (which you'll see is freed separately in expr_destroy).